### DISCUSSION TITLE
Mobile NFT Applications (self custodied)

### DISCUSSION THREAD

**Discussion by thelaneburch** [ordinals/ord#1599] [Ideas]
We need to figure out how to download the ledger on a mobile device with smaller hard drive capacity. This is the first step in using Bitcoin Core (or similar application) on a mobile device.

It's from there the community will be allowed to grow.

**Comment by casey** [comment]
Making ord light enough to run on a mobile device may require a lot of work, and possibly require an SPV mode, which is basically an open research problem at the moment. I would definitely encourage people to start noodling on:

1. What does a good security/ease of use trade-off look like for mobile use?
2. What does extending Bitcoin's SPV model to ordinals and inscriptions look like?
3. Can `ord` be used for this? Should we add support to this to `ord`? Or should we turn `ord` into a library / binary, so we can reuse library components w/a different client?

**Comment by gkrizek** [comment]
I think most existing mobile NFT applications out source most of this to a third party. There’s obviously trade offs with that, but just describing how most work. They call to an API to get inscription data and all that, but likely keep the wallet private keys on the phone to stay noncustodial. I agree with Casey doing it in a ‘purist’ way would be pretty challenging.
